.github/dependabot.yml

```
@@ -94,8 +95,6 @@ updates:
     include: scope
   ignore:
   - dependency-name: '*golang*'
```

I suspect if the `*golang*` is the problem here. Can we also update this to `golang` ? 
```suggestion
  - dependency-name: 'golang'
```

From the commit description of the #4559 , looks like dependabot tried to update `golang` dependency and for some reason, our regex `*golang*` did not match it.

---

I think the regex still should match: https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file#specifying-dependencies-and-versions-to-ignore

> Use to ignore updates for dependencies with matching names, optionally using `*` to match zero or more characters.

---

But I can see where we probably don't intend to match other dependencies that happen to contain `golang` anywhere in their name so I'll update this.